Cargo.toml

```diff
@@ -11,6 +11,10 @@ repository = "https://github.com/webrtc-rs/util"
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
+[features]
+default = []
+vnet = []
+
```

Comment:
can we add more other features in util crate? such as
buffer = []
conn = []
error = []
ifaces = []

Reply:
@rainliu Ok! Do ```fixed_big_int``` and ```replay_detector``` add too?

Reply:
lgtm.